assets/js/modules/tagmanager/components/common/UseSnippetSwitch.js

```diff
@@ -56,14 +64,22 @@ export default function UseSnippetSwitch() {
 				hideLabel={ false }
 			/>
 			<p>
-				{ useSnippet
-					? __(
-							'Site Kit will add the code automatically.',
-							'google-site-kit'
+				{ primaryContainerID === containerID
```

Comment:
The IB states:
>As the text for `UseSnippetSwitch` needs to be changed for the usage in the Setup flow, but remain the same for the current usage in the Settings flow, we are going to create two variants, `SetupUseSnippetSwitch` and `SettingsUseSnippetSwitch` in `assets/js/modules/tagmanager/components/common/` directory for use in each case.

However, we should have the same toggle snippet message for both the setup and settings. Hence, there is no need to create two different `UseSnippetSwitch` variants. Refer to [this](https://github.com/google/site-kit-wp/issues/4934#issuecomment-1098986186) and the following comments.